tasks/test_metadata.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: test-metadata
+spec:
+  results:
+    - name: test-event-type
+      description: "Stores information about the event type that indicates if a job is running for a Pull Request or Push event"
+    - name: pull-request-number
+      description: "Stores information about pull request number if exists"
+    - name: source-git-url
+      description: "Stores information about the source git url for the change. Can be from a fork or from the same repo."
+    - name: source-git-revision
+      description: "Stores information about the source git revision for the change."
+    - name: target-git-url
+      description: "Stores information about the target git repository."
+    - name: target-git-branch
+      description: "Stores information about the target git branch."
+    - name: component-name
+      description: "Stores information about the Konflux component that was built in case of the component-type Snapshot."
+    - name: component-container-image
+      description: "Stores information about the component's container that was built in case of the component-type Snapshot."
+  params:
+    - name: SNAPSHOT
+      description: The JSON string of the Snapshot under test
+  steps:
+    - name: test-metadata
+      image: quay.io/redhat-appstudio/konflux-test:stable
+      workingDir: /workspace
+      env:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: EVENT_TYPE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.labels['pac.test.appstudio.openshift.io/event-type']
+        - name: COMPONENT_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.labels['appstudio.openshift.io/component']
+        - name: PULL_REQUEST_NUMBER
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.labels['pac.test.appstudio.openshift.io/pull-request']
+        - name: TARGET_GIT_URL
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.annotations['pac.test.appstudio.openshift.io/repo-url']
+        - name: TARGET_GIT_BRANCH
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.annotations['pac.test.appstudio.openshift.io/branch']
+      script: |
+        #!/bin/sh
```

Comment:
Can we get also the Author of the commit?

Reply:
Yup, that should be available as standard afaik.

Reply:
Added the `sender` result, let me know if that works for you.